File: task-03/Elixir/Subtask3.ex
# Get the number from the user
n = String.to_integer(IO.gets("Enter a number: "))

# Top half of the diamond
for i <- 0..(n-1) do
  IO.puts String.duplicate(" ", n - i - 1) <> String.duplicate("*", 2 * i + 1)
end

# Bottom half of the diamond
for i <- (n-2)..0 do
  IO.puts String.duplicate(" ", n - i - 1) <> String.duplicate("*", 2 * i + 1)
end

File: task-03/Javascript/Subtask3.js
// Get the number from the user
const n = parseInt(prompt("Enter a number:"));

// Top half of the diamond
for (let i = 0; i < n; i++) {
  console.log(" ".repeat(n - i - 1) + "*".repeat(2 * i + 1));
}

// Bottom half of the diamond
for (let i = n - 2; i >= 0; i--) {
  console.log(" ".repeat(n - i - 1) + "*".repeat(2 * i + 1));
}

